src/components/Author/Author.test.tsx

```diff
@@ -51,14 +41,13 @@ describe('<Author /> component', () => {
       latest: {
         name: 'verdaccio',
         version: '4.0.0',
+        dist: { fileCount: 0, unpackedSize: 0 },
       },
+      _uplinks: {},
     };
 
-    // @ts-ignore
-    mockPackageMeta.mockImplementation(() => packageMeta);
-
-    const wrapper = mount(<Authors />);
-    expect(wrapper.html()).toEqual('');
+    const wrapper = mount(component(packageMeta));
+    expect(wrapper.html()).toBeNull();
```

Comment:
Previously the context `Consumer` was always rendered so this was an empty string. It changed to `null` since now if the context value is empty we return `null` (and don't have to render the `Consumer`).

Reply:
You might use https://github.com/testing-library/react-testing-library/  if you feel more confortable, as `enzyme` is also available. https://github.com/testing-library/react-testing-library/

Reply:
I did initially rewrite these with rtl. I stuck with enzyme in the end so the snapshots would still match in this PR and it would be easier to see that there are no regressions in the test cases.